dht_bootstrap.go

```diff
@@ -40,30 +42,24 @@ func init() {
 	}
 }
 
-// BootstrapConfig specifies parameters used bootstrapping the DHT.
-//
-// Note there is a tradeoff between the bootstrap period and the
-// number of queries. We could support a higher period with less
-// queries.
+// BootstrapConfig specifies parameters used for bootstrapping the DHT.
 type BootstrapConfig struct {
```

Comment:
cc @hsanjuan, @whyrusleeping, @frrist:

This will be an API breaking change, modifying the BootstrapConfig structure. This change is necessary to bring the DHT's bootstrapping logic in-line with the kademlia paper (properly refreshing _each_ bucket) instead of just making random queries in the dark.

Thoughts? Objections? Questions?

Reply:
Thank you for the ping, no objection from me.

Reply:
Sounds good to me.